core/src/test/java/com/coderanch/blackjack/HandTest.java

```diff
@@ -0,0 +1,324 @@
+/*
+ * Copyright (C) 2018 Coderanch.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.coderanch.blackjack;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import com.coderanch.blackjack.Card.Rank;
+import com.coderanch.blackjack.Card.Suit;
+
+import org.junit.experimental.theories.DataPoints;
+import org.junit.experimental.theories.FromDataPoints;
+import org.junit.experimental.theories.Theories;
+import org.junit.experimental.theories.Theory;
+import org.junit.runner.RunWith;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.equalTo;
```

Comment:
Put this import before the import to `is`.

Reply:
not sure what you mean

Reply:
I mean that within an import group, the imports should be sorted alphabetically, so put `equalTo` before `is`.